helm-charts/support/values.yaml

```diff
@@ -432,6 +432,10 @@ cryptnono:
         cpu: 5m
         memory: 100Mi
 
+  image:
+    # Brings in https://github.com/cryptnono/cryptnono/pull/33
```

Comment:
Maybe should have a FIXME note to update/change again once that PR is merged?

Reply:
@sgibson91 good idea, added!